content/en/docs/reference/node/cri-api-development-policies.md

```diff
@@ -0,0 +1,107 @@
+---
+content_type: "reference"
+title: Kubernetes feature development and Container runtimes
+weight: 10
+---
+
+The mechanics of a feature development that requires new CRI APIs is covered in
+documentation on CRI API [feature-development](https://github.com/kubernetes/cri-api?tab=readme-ov-file#feature-development).
+This article declares policies for developing new Kubernetes features
+that require CRI API changes. The goal of these policies is to ensure great user
+experience for people trying the new feature early, adopting it when it is
+enabled by default, and relying on it as a GA functionality.
+
+## Supported container runtimes
+
+Features and CRI API are supposed to be portable and generic and not limited to
+a specific container runtime. However at this moment we require every feature to
+work on two container runtimes: [Containerd](https://containerd.io/) and
+[CRI-O](https://cri-o.io/). These are two runtimes that are tested as part of a
+kubernetes development and release process. When this document refers to two
+container runtimes, it assumes both - Containerd and CRI-O. If any other
+container runtimes begin working actively with the Kubernetes community, this
+document will need to be updated.
```

Comment:
This sounds like details aimed at contributors, not end users.

Also also, [avoid using “we“](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)